packages/manager/modules/dbaas-logs/src/logs/detail/inputs/add/configure/logs-inputs-add-configure.controller.js

```diff
@@ -64,59 +53,36 @@ export default class LogsInputsAddConfigureCtrl {
           this.input.data,
         ),
     });
+
     this.previousTest = this.test;
 
     this.input.load().then(() => {
       this.previousTest.load();
-      if (this.configuration.engineType === this.LogsConstants.logstash) {
-        this.getEngine();
-      }
-    });
-  }
-
-  getEngine() {
-    this.loading.engine = true;
-    this.LogsInputsService.getInputEngines(this.serviceName).then((engines) => {
-      this.engine = find(engines, {
-        engineId: this.input.data.info.engineId,
-      });
-      this.loading.engine = false;
     });
   }
 
-  initFlowgger(configuration) {
-    this.configuration.flowgger.logFormat = configuration.logFormat;
-    this.configuration.flowgger.logFraming = configuration.logFraming;
-  }
-
   static findRowLength(str) {
-    const lines = str.split(/\r\n|\r|\n/);
-    return lines.length;
-  }
-
-  initLogstash(configuration) {
-    this.configuration.logstash.inputSection = this.CucControllerHelper.constructor.htmlDecode(
-      configuration.inputSection,
-    );
-    this.configuration.logstash.filterSection = this.CucControllerHelper.constructor.htmlDecode(
-      configuration.filterSection,
-    );
-    this.configuration.logstash.patternSection = this.CucControllerHelper.constructor.htmlDecode(
-      configuration.patternSection,
-    );
+    if (str) {
+      const lines = str.split(/\r\n|\r|\n/);
+      return lines.length;
+    }
+    return 5;
```

Comment:
Maybe you can move in constant file ?

Reply:
It's a static function so access to LogsConstants looks for me complicate. What is your recommendation?